netlify.toml

```diff
@@ -4,10 +4,10 @@
 # DO NOT REMOVE THIS (contact @kubernetes/sig-docs-leads)
 publish = "public"
 functions = "functions"
-command = "git submodule update --init --recursive --depth 1 && make non-production-build && npx -y pagefind --site public"
+command = "make non-production-build && npx -y pagefind --site public"
 
 [build.environment]
-NODE_VERSION = "20.17.0"
+NODE_VERSION = "20.11.0"
```

Comment:
Calling out a downgrade here

Reply:
Oh, that shouldn't be there. Reverting this change.